### PR TITLE
Update README.org

### PR DESCRIPTION
**Pull request by eric-lac**
Fixed spelling mistakes and grammatical errors in the README.org

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -179,7 +179,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
    # ATTENTION IF YOU ARE CONTRIBUTOR: This parent category "Interface
    # Enhancement" is frequently abused. Please DO NOT casually place
    # package in this parent category, unless the package is indeed not
```
Comment (agsdot) [inline README.org:181]: @eric-lac, can you update the PR further?

This section looks like more grammar changes are needed as well. Can you give a run over and reword this section too?

Something like
```
Please DO NOT casually place package in this parent category, unless the package is indeed not belong to any other subcategory, or other category.
```

to
```
Please DO NOT casually place packages in this parent category, unless the package indeed does not belong to any other subcategory or other category.
```

The package vs packages, singular vs plural, and how the diction flows could be improved. Thanks for contributing improvements here.

Comment (agsdot) [reply README.org:181]: Use whatever creative license / creativity you want to use here. Thanks!

File: README.org [hunk 2]
```diff
@@ -351,7 +351,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
 
    - [[https://www.emacswiki.org/emacs/DeleteSelectionMode][delete-selection-mode]] - =[built-in]= Auto delete selected region when typing, just like most of other modern text editors.
    - [[https://github.com/soutaro/hungry-delete.el][hungry-delete]] - Delete an entire block of whitespace at point.
-   - [[https://github.com/hrehfeld/emacs-smart-hungry-delete][smart-hungry-delete]] -  Delete whitespace between words, parenthesis and other delimiters in a (not very) smart way.
+   - [[https://github.com/hrehfeld/emacs-smart-hungry-delete][smart-hungry-delete]] -  Delete whitespace between words, parenthesis, and other delimiters in a (not very) smart way.
```
Comment (agsdot) [inline README.org:354]: The original description at https://github.com/hrehfeld/emacs-smart-hungry-delete contains the prior wording.

I'm not sure the Oxford Comma should/needs to stylistically be edited into this entire README.org file.

File: README.org [hunk 3]
```diff
@@ -527,7 +527,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
     - [[https://github.com/Lindydancer/cmake-font-lock][cmake-font-lock]] - Enhanced font-lock rules for CMake.
     - [[https://github.com/abo-abo/function-args][function-args]] - visual CEDET enhancements for C++.
     - [[https://www.gnu.org/software/emacs/manual/html_node/ebrowse/index.html][Ebrowse]] - =[built-in]= A C++ class browser.
-    - [[https://github.com/atilaneves/cmake-ide/][cmake-ide]] - Configures other packages to consider compile options like include paths from cmake projects to improve e.g. autocompletion.
+    - [[https://github.com/atilaneves/cmake-ide/][cmake-ide]] - Configures other packages to consider compile options like include paths from cmake projects to improve e.g. auto-completion.
```
Comment (agsdot) [inline README.org:530]: 👍 

Another thought, the parent repo https://github.com/atilaneves/cmake-ide/ has the same typo / spelling error. 

I'm fine with this modification, adjusting it or not adjusting it.

File: README.org [hunk 4]
```diff
@@ -549,7 +549,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
     - [[https://www.emacswiki.org/emacs/yari.el][yari]] - Yet Another RI interface for Emacs.
     - [[https://github.com/dgutov/robe][robe]] - Code navigation, documentation lookup and completion for Ruby.
     - [[https://github.com/rubocop/rubocop][rubocop]] - A Ruby static code analyzer, based on the community Ruby style guide.
-    - [[https://github.com/ainame/motion-mode][motion-mode]] - A package to provide emacs' major mode for RubyMotion enviroment.
+    - [[https://github.com/ainame/motion-mode][motion-mode]] - A package to provide emacs' major mode for RubyMotion environment.
```
Comment (agsdot) [inline README.org:552]: Thanks for the spelling error typo catch @eric-lac ! Much appreciated 😃

File: README.org [hunk 5]
```diff
@@ -569,7 +569,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
 **** Scheme
 
      - [[https://www.neilvandyke.org/quack/][Quack]] - Enhanced Emacs Support for Editing and Running Scheme Code.
-     - [[http://www.nongnu.org/geiser/][Geiser]] - Intergrated development with Guile and Racket.
+     - [[http://www.nongnu.org/geiser/][Geiser]] - Integrated development with Guile and Racket.
```
Comment (agsdot) [inline README.org:572]: 👍

File: README.org [hunk 6]
```diff
@@ -636,7 +636,7 @@ Above all, enjoy using Emacs. The community, that more than anything, makes Emac
 
 *** Java
 
-    - [[https://github.com/emacs-eclim/emacs-eclim][emacs-eclim]] - An Eclipse plugin which exposes Eclipse features through a server interface.
+    - [[https://github.com/emacs-eclim/emacs-eclim][emacs-eclim]] - An Eclipse plugin that exposes Eclipse features through a server interface.
```
Comment (agsdot) [inline README.org:639]: The original repo (which points towards lsp-java now...) uses the same previous grammar. 

I'm fine with keeping the new changes or keeping the old wording. Either or. Thanks for the catch.

File: README.org [hunk 7]
```diff
@@ -852,7 +852,7 @@ External Guides:
    - [[https://github.com/jwiegley/git-undo-el][git-undo]] - A command for Emacs to regress or "undo" a region back through its Git history.
 
 #+BEGIN_QUOTE
-For additional git related emacs packages to use or to get inspiration from, take a look at the following resource: [[https://github.com/tarsius/git-elisp-overview]].
+For additional git-related emacs packages to use or to get inspiration from, take a look at the following resource: [[https://github.com/tarsius/git-elisp-overview]].
```
Comment (agsdot) [inline README.org:855]: 👍

File: README.org [hunk 8]
```diff
@@ -958,7 +958,7 @@ For additional git related emacs packages to use or to get inspiration from, tak
 
 ** PDF
 
-   - [[https://github.com/vedang/pdf-tools][PDF Tools]] - major mode for rendering PDF files, much better than DocView, and has much richer set of features.
+   - [[https://github.com/vedang/pdf-tools][PDF Tools]] - major mode for rendering PDF files, much better than DocView, and has a much richer set of features.
```
Comment (agsdot) [inline README.org:961]: Good catch. Thanks for adding the article (a/an/the, etc)

**Comment by agsdot** [comment]
@eric-lac , thanks for catching some of the punctuation issues (catching the Oxford commas) and the areas in which hyphens needed to be inserted.

One thing though, can these changes be made uniform across the entire README.org file? Can you make the grammar check for the entire file?

Early on, I put in trailing periods to entries, to honestly, just make the grammar of each entry consistent. Something the same can be done for the oxford comma, hyphens, and the like.

**Comment by agsdot** [comment]
One thing I wonder about though, some of the package descriptions are copy pasted verbatim from their parent project repos. 

Should those wordings always be modified, or can they be "let to slide"? Just a thought. 

Thanks for your insights @eric-lac . Much appreciated.

File: README.org [hunk 9]
```diff
@@ -1203,15 +1203,15 @@ For additional git related emacs packages to use or to get inspiration from, tak
...
+*Your contributions are always welcome!* Please submit a pull request or create an issue to add a new package, library, or software to the list.
 
 Before contributing, please read this tiny guideline:
 
 ** The Order of Items
 
 *Please don't rearrange the package ordering without any reason!* The items should be sorted by its popularity ( /roughly, because it's impossible to have a precise standard./ e.g. _most people would use it_; _recommended for every newbie_...), instead of your personal preference.
 
-For example, =Evil= is obviously not a package that every Emacser needs. So please don't move it onto the top of its category.
+For example, =Evil= is obviously not a package that every Emacser needs. So please don't move it to the top of its category.
```
Comment (agsdot) [inline README.org:1214]: 👍

File: README.org [hunk 10]
```diff
@@ -636,7 +636,7 @@ Above all, enjoy using Emacs. The community, more than anything, makes Emacs a g
 
 *** Java
 
-    - [[https://github.com/emacs-eclim/emacs-eclim][emacs-eclim]] - An Eclipse plugin that exposes Eclipse features through a server interface.
+    - [[https://github.com/emacs-eclim/emacs-eclim][emacs-eclim]] - An Eclipse plugin which exposes Eclipse features through a server interface.
```
Comment (eric-lac) [inline README.org:639]: Restored the original repo wording

File: README.org [hunk 11]
```diff
@@ -527,7 +527,7 @@ Above all, enjoy using Emacs. The community, more than anything, makes Emacs a g
     - [[https://github.com/Lindydancer/cmake-font-lock][cmake-font-lock]] - Enhanced font-lock rules for CMake.
     - [[https://github.com/abo-abo/function-args][function-args]] - visual CEDET enhancements for C++.
     - [[https://www.gnu.org/software/emacs/manual/html_node/ebrowse/index.html][Ebrowse]] - =[built-in]= A C++ class browser.
-    - [[https://github.com/atilaneves/cmake-ide/][cmake-ide]] - Configures other packages to consider compile options like include paths from cmake projects to improve e.g. auto-completion.
+    - [[https://github.com/atilaneves/cmake-ide/][cmake-ide]] - Configures other packages to consider compile options like include paths from cmake projects to improve e.g. autocompletion.
```
Comment (eric-lac) [inline README.org:530]: Restored back to original repo (no hyphen)

File: README.org [hunk 12]
```diff
@@ -351,7 +351,7 @@ Above all, enjoy using Emacs. The community, more than anything, makes Emacs a g
 
    - [[https://www.emacswiki.org/emacs/DeleteSelectionMode][delete-selection-mode]] - =[built-in]= Auto delete selected region when typing, just like most of other modern text editors.
    - [[https://github.com/soutaro/hungry-delete.el][hungry-delete]] - Delete an entire block of whitespace at point.
-   - [[https://github.com/hrehfeld/emacs-smart-hungry-delete][smart-hungry-delete]] -  Delete whitespace between words, parenthesis, and other delimiters in a (not very) smart way.
+   - [[https://github.com/hrehfeld/emacs-smart-hungry-delete][smart-hungry-delete]] -  Delete whitespace between words, parenthesis and other delimiters in a (not very) smart way.
```
Comment (eric-lac) [inline README.org:354]: Restored back to the original repo (no oxford comma)

**Comment by agsdot** [comment]
Thanks for adding in this PR @eric-lac . Your contributions are greatly appreciated!

**Comment by agsdot** [comment]
Squashed and merged in the PR.

**Pull request merged**